docs/source/tutorials/filtsmooth/linear_bayesian_filtering_smoothing.ipynb

```diff
@@ -0,0 +1,4552 @@
+{
```

Comment:
<p>"Only the dynamics model changes' -- well, the data does too?</p><p> </p><p>"linear, time-invariant stochastic differential equation (SDE)".</p><p> </p><p> </p><p>Please remind me that we should discuss whether we want to talk about "Wiener processes", or "Brownian motion", and which letters we use in the notation. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/292/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>I think talking about Brownian Motion might be more consistent, cf. <code>IBM</code>.  What do you think?</p>

Reply:
<p>in this case yes. Though, on a side note, I personally prefer Wiener process (as this is also how I write my papers about this stuff, and AFAIK, nathanael does too), but this is beyond this PR. There is a discussion in the ODE notation about this issue, so we can continue there,. </p>

Reply:
<p>Agreed.</p>